src/Game/GamePage.js

```diff
@@ -28,32 +28,32 @@ const GamePage = (props) => {
       const playerHp = data.map((playerData) => {
         return { name: playerData.name, Hitpoints: playerData.Hitpoints };
       });
-      setTotalPlayersHp(playerHp);
+      settotalPlayerData(playerHp);
```

Comment:
```suggestion
      setTotalPlayerData(playerHp);
```
have to fix all instances of this